src/lib/apis/auths/index.ts

```
@@ -82,55 +82,139 @@ export const updateAdminConfig = async (token: string, body: object) => {
+					rawText.trim().charAt(0),
+				);
+				console.error(
+					"[DEBUG] getSessionUser: Raw text first 20 chars codes:",
+					rawText
+						.substring(0, 20)
+						.split("")
+						.map((c) => c.charCodeAt(0)),
+				);
+				throw new Error("Server returned an invalid response format");
+			}
+		} catch (parseError: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)